documentation/src/main/asciidoc/topics/protostream.adoc

```diff
@@ -0,0 +1,163 @@
+[[protostream]]
+= ProtoStream
+
+link:https://github.com/infinispan/protostream[ProtoStream] is a serialization library based on link:https://developers.google.com/protocol-buffers[Protocol Buffers].
+The link:https://github.com/infinispan/protostream[ProtoStream] library, is responsible for encoding/decoding Java objects
+to/from the Protobuf format and is used extensively throughout {brandname}.
+
+This chapter provides a brief overview of the Protobuf and ProtoStream concepts, before detailing how ProtoStream can be
+used to serialize your Java objects.
+
+== Concepts
+=== .proto Files
+NOTE: Protocol Buffers is a broad subject, we will not detail it here in great detail, so please consult the
+link:https://developers.google.com/protocol-buffers/docs/overview[Developer Guide] for an in-depth
+explanation of the encoding format and best practices.
+
+Protocol Buffers, Protobuf for short, provide a platform independent encoding format that utilises .proto schema files to
+provide a structured representation of entities that can be easily evolved over time whilst maintaining backwards compatibility.
+
+Protobuf is all about structured data, so the first thing todo is to define the structure of your data. This is
+accomplished by declaring Protobug message types in .proto files, as shown in the example below.
```

Comment:
s/Protobug/Protobuf/

Although I do think I want to learn more about protobugs too :)